nni/compression/pytorch/quantization_speedup/trt_pycuda.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import pycuda.driver as cuda
+import pycuda.autoinit
+import tensorrt as trt
+
+pycuda.autoinit
```

Comment:
maybe remove?

Reply:
This have to be kept because `import pycuda.autoinit` is necessary here otherwise pycuda would not be ready and error would be raised. But itself will not be used in following code which is not allowed in python test pipeline. So I choose to put this sentence here.

Reply:
better to use comment to escape pylint for that line